### ISSUE TITLE
ternary plots for piper diagram

### ISSUE THREAD

**Issue by jkall** [jkall/qgis-midvatten-plugin#103]
this library might be useful to enable ternary plots in the piper diagrams:
https://github.com/marcharper/python-ternary

**Comment by jkall** [comment]
Or else go for the freewat integration:
https://github.com/jkall/qgis-midvatten-plugin/issues/264

**Comment by inkenbrandt** [comment]
This library could be augmented for piper ternarys: https://github.com/inkenbrandt/Peeters_Piper

**Comment by jkall** [comment]
> This library could be augmented for piper ternarys: https://github.com/inkenbrandt/Peeters_Piper

Yes, this is an interesting library and it is probably less work than the other alternatives. 

Could be added as an additional  feature in plot menu "Plot - Ternary Piper Diagram". Probably possible to reuse parts of the "get_data_..." method in current PiperPlot class.

**Comment by jkall** [comment]
The feature "Piper diagram version 2" resolves this issue. 

But the plot proportions (not size) should be fixed so that the ternary plots are equilateral triangles. 
(The current version results in triangles where base > sides which may result in misleading visual presentation of data.)
